1_16/pom.xml

```diff
@@ -10,7 +10,7 @@
         <version>1.9.3-SNAPSHOT</version>
     </parent>
 
-    <artifactId>anvilgui-1_16_R1</artifactId>
+    <artifactId>anvilgui-1_16_1</artifactId>
```

Comment:
The `artifactId` should be `1_16` instead of `1_16_1`, to match the module and Wrapper class name